apps/web/src/pages/dashboard/index.tsx

```
@@ -1,12 +1,54 @@
 
   return (
-    <div>
-      <h1>안녕하세요 {user?.username} 님!</h1>
+    <div className="bg-gray-5 px-16 py-40 md:p-80">
+      <section className="mb-80 flex flex-col gap-16">
+        <h1 className="text-28-700 text-gray-100">내 회의</h1>
+        {/* line */}
+        <span className="border-gray-100-opacity-10 h-1 w-full border-b-[1px]" />
+        {userReservations?.map((reservation: Reservation) => (
+          <ReservationCard reservation={reservation} />
+        ))}
```

Biome

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)